connectivity/FEATURE_BLE/include/ble/common/ChainableEventHandler.h

```diff
@@ -69,7 +69,7 @@ class ChainableEventHandler
     /**
      * Remove an EventHandler previously added with addEventHandler.
      *
-     * @param[in] event_handler Pointer to event handler to remove
+     * @param[in] target Pointer to event handler to remove
```

Comment:
maybe it's better to change the argument name to event_handler - makes more sense

Reply:
I should have fixed the function declaration, not the comment. Thanks